hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/rollback/ListingBasedRollbackHelper.java

```diff
@@ -116,14 +116,31 @@ public ListingBasedRollbackHelper(HoodieTableMetaClient metaClient, HoodieWriteC
                   .withDeletedFileResults(filesToDeletedStatus).build());
         }
         case APPEND_ROLLBACK_BLOCK: {
+          // collect all log files that is supposed to be deleted with this rollback
+          String baseCommit = rollbackRequest.getLatestBaseInstant().get();
```

Comment:
`baseInstantTime`

Reply:
all of this code is pretty much replaced by FSUtils.getAllLogFiles()?

Reply:
Existing FSUtils.getAllLogFiles() expects fileId to be passed in. Hence have created a new method in FSUtils which lists all files disregarding fileId, but later filters for basecommit passed in.